tests/helpme/test_integrate.py

```diff
@@ -178,15 +178,23 @@ def test_function_integration(np_rng):
     col1_factor = 2
 
     def volume_function(x):
-        vf = np.ones((x.shape[0], 2))
+        vf = np.ones((len(x), 2))
         # scale it with a factor to get a different value
         vf[:, 1] = col1_factor
```

Comment:
Also a factor would assume a multiplication and not just the value. In this special case, the result is the same, but it might be a little weird for someone to read it and see that it just overwrites the value instead of multiplying it. So `col2_value` would be more accurate.

Reply:
I thought col starts at 0?